cmd/infracost/run.go

```diff
@@ -50,6 +50,10 @@ func addRunFlags(cmd *cobra.Command) {
 }
 
 func runMain(cmd *cobra.Command, runCtx *config.RunContext) error {
+	if runCtx.Config.IsSelfHosted() && runCtx.Config.EnableDashboard {
+		ui.PrintWarning(cmd.ErrOrStderr(), "The dashboard is part of Infracost's hosted services. Contact hello@infracost.io for help.")
+	}
```

Comment:
@aliscott FYI, tiny change since your review: moved this message to the top to appear before terraform spinners.

Reply:
👍